lib/ansible/modules/cloud/amazon/elb_application_lb_facts.py

```diff
@@ -50,6 +50,13 @@
     names:
       - elb1
       - elb2
+
+# Gather facts about specific ALB
```

Comment:
Do you really mean ALB here (Application Load Balancer?) or ELB? or LB?

Reply:
I tested it with Application LB, Network LB and Classic LB. And it works only with ALB and NLB.